### DISCUSSION TITLE
Hot to inject custom actions in afterCreate()

### DISCUSSION THREAD

**Discussion by hrisitjansiwb** [filamentphp/filament#6547] [Help]
I have a custom action that I want to execute once the record is created/updated - for example in the afterCreate/afterUpdate methods.

I can c reate new instance of my action using $action = app()->make(ActionName::class) and call it without any issues. I was wondering if there is a way to inject an Interface and call the action within hooks?

**Comment by danharrin** [comment]
We don't use the container to call hooks, no. But you could absolutely PR that in, would be a 1 line change!
